src/reducers/index.js

```diff
@@ -12,9 +12,17 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
+      return {
+        ...state,
+        count: initialState.count++
```

Comment:
So you probably noticed that there is a one click delay whenever you try to update the count. the reason that is happening is because of this line. 2 things are happening here. You are setting the count to the value of count in the initial state, and THEN you are incrementing the initialstate count. 

So you shouldn't be using the intitialState at all in your switch statement. The initialState object is just the default object that sets your initial redux store. Up on line 12, that `state = initialState` just means if there is no state being passed in, as in the case of createStore in the index.js file, then initialState is used. Any action run after the store is created will have the redux store passed in as state. So you should be using that.